README.md

```diff
@@ -1,8 +1,8 @@
 <p align="center"><a href="https://darkreader.org" target="_blank" rel="noreferrer noopener"><img width="250" alt="Dark Reader's mascot" src="https://raw.githubusercontent.com/darkreader/darkreader.github.io/master/images/darkreader-mascot.svg"></a></p>
 <p align="center">Dark Reader <strong>analyzes</strong> web pages and aims to <strong>reduce the eyestrain</strong> while you browse the web.</p>
 <br/>
-<p align="center"><a rel="noreferrer noopener" href="https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh/">Chrome Web Store</a> | <a rel="noreferrer noopener" href="https://addons.mozilla.org/en-US/firefox/addon/darkreader/">Firefox Add-ons</a> | <a rel="noreferrer noopener" href="https://microsoftedge.microsoft.com/addons/detail/dark-reader/ifoakfbpdcdoeenechcleahebpibofpc/">Edge Addons</a> | <a el="noreferrer noopener" href="https://addons.thunderbird.net/en-US/thunderbird/addon/darkreader">Thunderbird</a>
-    
+<p align="center"><a rel="noreferrer noopener" href="https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh/"><img alt="Chrome Web Store" src="https://img.shields.io/badge/Chrome%20Web%20Store-141e24.svg?&style=flat-square&logo=google-chrome&logoColor=white"></a><a rel="noreferrer noopener" href="https://addons.mozilla.org/en-US/firefox/addon/darkreader/"><img alt="Firefox Add-ons" src="https://img.shields.io/badge/Firefox%20Addons-141e24.svg?&style=flat-square&logo=firefox-browser&logoColor=white"></a><a rel="noreferrer noopener" href="https://microsoftedge.microsoft.com/addons/detail/dark-reader/ifoakfbpdcdoeenechcleahebpibofpc/"><img alt="Edge Addons" src="https://img.shields.io/badge/Edge%20Addons-141e24.svg?&style=flat-square&logo=microsoft-edge&logoColor=white"></a><a el="noreferrer noopener" href="https://addons.thunderbird.net/en-US/thunderbird/addon/darkreader"><img alt="Thunderbird" src="https://img.shields.io/badge/Thunderbird-141e24.svg?&style=flat-square&logo=thunderbird&logoColor=white"></a>
```

Comment:
```suggestion
<p align="center"><a rel="noreferrer noopener" href="https://chrome.google.com/webstore/detail/dark-reader/eimadpbcbfnmbkopoojfekhnkhdbieeh/"><img alt="Chrome Web Store" src="https://img.shields.io/badge/Chrome%20Web%20Store-141e24.svg?&style=flat-square&logo=google-chrome&logoColor=white&logoWidth=45"></a>  <a rel="noreferrer noopener" href="https://addons.mozilla.org/en-US/firefox/addon/darkreader/"><img alt="Firefox Add-ons" src="https://img.shields.io/badge/Firefox%20Addons-141e24.svg?&style=flat-square&logo=firefox-browser&logoColor=white&logoWidth=45"></a>  <a rel="noreferrer noopener" href="https://microsoftedge.microsoft.com/addons/detail/dark-reader/ifoakfbpdcdoeenechcleahebpibofpc/"><img alt="Edge Addons" src="https://img.shields.io/badge/Edge%20Addons-141e24.svg?&style=flat-square&logo=microsoft-edge&logoColor=white&logoWidth=45"></a>  <a el="noreferrer noopener" href="https://addons.thunderbird.net/en-US/thunderbird/addon/darkreader"><img alt="Thunderbird" src="https://img.shields.io/badge/Thunderbird-141e24.svg?&style=flat-square&logo=thunderbird&logoColor=white&logoWidth=45"></a>
```
Make it look a little bit bigger and add the 2 spaces between them to distance them.
![image](https://user-images.githubusercontent.com/25481501/110699155-dc1cb300-81ee-11eb-81f3-af83ff23fd1b.png)